### PR TITLE
iannix: 2016-01-31 -> 0.9.20-b

### PR DESCRIPTION
**Pull request by freezeboy**
###### Motivation for this change

Use an official release. This version still doesn't allow yet to migrate to Qt5.15

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/iannix/default.nix [hunk 1]
```diff
@@ -20,11 +21,11 @@ mkDerivation {
...
-    description = "Graphical open-source sequencer,";
+  meta = with lib; {
+    description = "Graphical open-source sequencer";
     homepage = "https://www.iannix.org/";
-    license = stdenv.lib.licenses.lgpl3;
-    platforms = stdenv.lib.platforms.linux;
-    maintainers = [ stdenv.lib.maintainers.nico202 ];
+    license = licenses.lgpl3;
+    platforms = platforms.linux;
+    maintainers = [ maintainers.nico202 ];
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/iannix/default.nix:29]:
```suggestion
    maintainers = with maintainers; [ nico202 ];
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Can you rebase this branch on master to fix the eval error?

**Comment by nico202** [comment]
@freezeboy Would you mind maintaining this package? I'm running NixOS only on an headless server now

**Comment by RaghavSood** [comment]
Result of `nixpkgs-review pr 104447` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>iannix</li>
  </ul>
</details>

**Pull request merged**